test/specs/web-messenger.spec.js

```diff
@@ -203,7 +203,7 @@ describe('WebMessenger', () => {
                                     appId: hat()
                                 };
 
-                                return WebMessenger.init(props).then(() => {
+                                return WebMessenger.init(props).catch(() => {
```

Comment:
nit: `.init(props).should.be.rejected.then(() => { })` to make sure that a successful init doesn't pass this test (and the others below)